src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image-container">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="Image product imac"
+        >
+      </div>
+
+      <h1 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__review-text">Reviews: 5</p>
+      </div>
+
+      <div class="card__price-block">
+        <p>Price:</p>
+        <span class="card__price">$2,199</span>
```

Comment:
```suggestion
      <div class="card__price">
        <p class="card__price-text>Price:</p>
        <span class="card__price-value">$2,199</span>
```
consider better naming.
move all styles which need only for Price text to .card__price-text element.